openapi-contracts/src/test/java/com/sinch/sdk/domains/verification/models/dto/v1/VerificationReportDtoTest.java

```diff
@@ -0,0 +1,90 @@
+package com.sinch.sdk.domains.verification.models.dto.v1;
+
+import com.adelean.inject.resources.junit.jupiter.GivenJsonResource;
+import com.adelean.inject.resources.junit.jupiter.TestWithResources;
+import com.sinch.sdk.BaseTest;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+@TestWithResources
+public class VerificationReportDtoTest extends BaseTest {
+
+  @GivenJsonResource("/domains/verification/v1/VerificationReportCalloutResponseDto.json")
+  VerificationResponseDto loadedVerificationCalloutDto;
+
+  public static VerificationResponseDto expectedVerificationCalloutDto =
+      new VerificationResponseDto()
+          .id("the id")
+          .method(VerificationMethodDto.CALLOUT)
+          .status("FAIL")
+          .reason("Fraud")
+          .callComplete(true)
+          .reference("my reference")
+          .price(
+              new VerificationResponsePriceDto(
+                  new VerificationPriceInformationDto()
+                      .verificationPrice(
+                          new VerificationPriceInformationVerificationPriceDto(
+                              new MoneyDto()
+                                  .currencyId("verificationPrice currency id")
```

Comment:
The specification saying that the currency id is a "ISO 4217 currency code", you may want to use a real one in case we implement a type validation at some point

Reply:
Good catch. I'll create a ticket and having to use it for Numbers API too 👍

Reply:
For reference: https://docs.oracle.com/javase/8/docs/api/java/util/Currency.html and ticket created (DEVEXP-206)